### PR TITLE
Improve API documentation on environment variables

### PR DESCRIPTION
**Pull request by sembrestels**

### CODE REVIEW

File: api/src/orgs/schema.js [hunk 1]
```diff
@@ -10,6 +10,8 @@ export default `
     kit: String
     # The total amount of assets under management (in Dai) for the organisation.
     aum: Float
+    # The total amount of ANT for the organisation.
+    ant: Float
```
Comment (sembrestels) [inline api/src/orgs/schema.js]: Not sure if we want to expose `ANT` in the API, but it already appears in the [resolver](https://github.com/1Hive/apiary/blob/master/api/src/orgs/resolver.js#L23).

Comment (onbjerg) [reply api/src/orgs/schema.js]: Can you make this a separate PR? I think I missed it in #144

Comment (sembrestels) [reply api/src/orgs/schema.js]: It solves:
`UnhandledPromiseRejectionWarning: Error: Organisation.ant defined
in resolvers, but not in schema`

Which is not really related with this pull request, but isn't related with #144 as it is a bug by itself.

Do you want me to do a separate pull request with only that fix?

Comment (sembrestels) [reply api/src/orgs/schema.js]: I moved the second commit to #150.

**Pull request merged**